scripts/variantstore/beta_docs/run-your-own-samples.md

```diff
@@ -0,0 +1,176 @@
+
+---
+
+**Warning:**        
+The workflow in the GVS beta workspace is configured based on the format of the TSV file. To avoid reconfiguring the workflow, do **not** rearrange or rename the columns in the TSV file.
+
+---
+
+7. Follow the steps in [How to use the Data Uploader](https://support.terra.bio/hc/en-us/articles/4419428208411) to **upload your data and TSV file** to Terra.
+
+## Run the workflow
+
+Now that your samples are loaded into data table in Terra, it’s time to setup and run the GVS workflow! The workflow is configured to call inputs from and write outputs to the sample_set table.
+
+1. **Select the workflow** from the Workflows tab.
+1. In the configuration page, select the **sample_set entity** in Step 1.
+1. Create a new sample_set dataset in Step 2.
+    1. Select **Create a new sample_set from selected samples**.
+    1. Click on the **arrow next to the tickbox** in the header row of the table.
+    1. Select All in the dropdown menu that appears to select all of the samples.
+    1. If you do **not** want to include the example data in your callset, **click on the tickbox** next to each of the 10 samples to deselect them.
+    1. At the bottom of the page, **name your new sample_set**. This will appear in the sample_set data table.
+    1. Click **OK** to create the sample_set.
+1. Configure the workflow inputs.
+    1. Enter a **name for the callset** as a string with the format “*CALLSET_NAME*” for the `callset_identifier` variable. This string is used as to name several variables and files and should begin with a letter. Valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”.
+    1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable.
+    1. Enter the name of the **GCP project** that holds the BigQuery dataset as a string with the format “*PROJECT_NAME*” for the `project_id` variable.
+1. **Save** the workflow configuration.
+1. **Run** the workflow.
+
+### Adding additional data to the BigQuery dataset
+The workflow imports GVCFs to the BigQuery dataset. The example GVCFs for the workspace are listed as a set in the workspace sample_set table. Multiple sample sets may be added to the same BigQuery dataset to appear in the same callset, but the workflow only runs on one sample set at a time. 
+
+If you run the workflow with the example data pre-loaded into the workspace, and then load additional data into that same dataset, the example data will be in your callset. Be sure to make a new dataset for your own data if you run the example data (unless you want the example data in there too–the more the merrier!).
+
+### Important configuration notes
+
+By default, the workflow is set up to write outputs to the workspace Google bucket. If you want to write the outputs to a different cloud storage location, you can specify the cloud path in the `extract_output_gcs_dir` optional input in the workflow configuration. 
+
```

Comment:
```suggestion
To access the interval lists for all of the output VCF shards:
Click into your job from the job history tab. This page for your job will have a table with metadata from your job in it (such as workflow ID and the Run Cost) and the table will have a Links column.
The icons/links in that column have additional job information.
Job Manager
Workflow Dashboard
Execution directory
There are two ways to get the interval lists from here. You can either use the UI in the Job Manager, or navigate for the correct path from the Execution directory.
Using the Job Manager:
If you prefer to use the job manager, click on the job manager link from the links column. From the List View tab, click into the GvsUnified page. This page's List View tab will show a list of all the sub-workflows in your job. From there, click into the GvsExtractCallset sub-workflow page. This page's List View tab will show a list of all tasks in the extract sub-workflow. You are interested in the SplitIntervals task which should be the 8th task. In the SplitIntervals row, find the icon in the outputs column and click on it. A popover will load, which will contain an array of the interval_files. These are the paths for the interval lists.
If the popover does not load, you can click into the SplitIntervals Execution directory (which is in the Links column of this table).
Using the Execution directory:
Your jobs Execution directory contains artifacts from your job, including logs and the interval lists.
You can access the interval lists by navigating to the directory for the SplitIntervals task inside the GvsExtractCallset sub-workflow.
Once you have clicked on the link to the Execution directory, it will drop you into your workflow bucket.
From there, click into the call-GvsUnified --> GvsUnified --> <unified job id (this will be the only option, so no lookup is needed)>
Then select the extract sub-workflow: call-GvsExtractCallset --> GvsExtractCallset --> <extract job id (only option--no lookup needed)>
Finally, select the SplitIntervals task: call-SplitIntervals and find the glob-<> directory. Inside that directory are the interval lists for the call set.
The interval list paths will look something like this:
gs://<workspace bucket id>/<submission id>/GvsJointVariantCalling/<workflow id>/call-GvsUnified/GvsUnified/<unified job id (only option--no lookup needed)>/call-GvsExtractCallset/GvsExtractCallset/<extract job id (only option--no lookup needed)>/call-SplitIntervals/glob-<task id>/0000000000-<callset id>.vcf.gz.interval_list
(Note: there will also be a glob-<task id> file with a list of interval lists, but not their paths)
Note:
The interval lists are named consistently with the vcfs: 00000000.vcf.gz.interval-list will go with 00000000.vcf.gz and 00000000.vcf.gz.tbi 
```